certificate/views.py

```diff
@@ -136,7 +140,7 @@ def verification(serial, _type):
                         ('Days', '11 June'), ('Year', year)])
                 elif purpose == 'SciPy India':
                     detail = OrderedDict([('Name', name), ('Event', purpose),
-                        ('Days', '14 - 16 December'), ('Year', year)])
+                        ('Days', '10 - 11 December'), ('Year', year)])
```

Comment:
After modifying verification, kindly check verification for older SciPy conferences.